modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -57,7 +57,7 @@
 	fire_sound = 'sound/weapons/gun/smg/shot_suppressed.ogg'
 
 /obj/projectile/beam/laser/hardlight_bullet/vintorez
-	damage = 27 // All security armory firearms need to do, at minimum, laser gun damage.
+	damage = 30 // 11/20/21: Increased from 27 to 30 to improve underperforming guns.
```

Comment:
this thing is still just terrible, average of 9s to down (compared to a bulldog with buckshots 5)

all following are done on local and live, vs unarmored humans

Reply:
these dropped in the wrong order, but they were tested on local and live, all 4